modules/tls-profiles-kubernetes-configuring.adoc

```diff
@@ -0,0 +1,117 @@
+// Module included in the following assemblies:
+//
+// * security/tls-profiles.adoc
+
+[id="tls-profiles-kubernetes-configuring_{context}"]
+= Configuring the TLS security profile for the control plane
+
+To configure a TLS security profile for the control plane, edit the `APIServer` custom resource (CR) to specify a predefined or custom TLS security profile. Setting the TLS security profile in the `APIServer` CR propagates the setting to the following control plane components:
+
+* Kubernetes API server
+* OpenShift API server
+* OpenShift OAuth API server
+* OpenShift OAuth server
+
+If a TLS security profile is not configured, the default TLS security profile is `Intermediate`.
+
+[NOTE]
+====
+The default TLS security profile for the Ingress Controller is based on the TLS security profile set for the API server.
+====
+
+.Sample `APIServer` CR that configures the `Old` TLS security profile
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: APIServer
+ ...
```

Comment:
So the canonical name for this is always going to be `cluster`, so I think it's helpful if we include that always:

```
apiVersion: config.openshift.io/v1
kind: APIServer
metadata:
  name: cluster
```

Reply:
@jboxman - Thanks a lot for all the comments. This PR is basically a selective back-porting of content that's already in 4.8 to 4.7 and 4.6. In order to maintain consistency, @bergerhoffer will be looking into all these suggested changes for 4.8 using a separate PR and getting them reviewed.